contents/handbook/company/story.md

```diff
@@ -56,3 +56,11 @@ We've now people in 10 countries around the world, and still feel like it's day
 
 Everyone takes a mandatory two weeks off over Christmas to relax.
 
+## $15m Series B - June, 2021
+
+We raised a $15m Series B [a little ahead of schedule](https://posthog.com/blog/why-we-raised-a-15m-series-b-ahead-of-schedule), led by existing investors Y Combinator. 
```

Comment:
@piemets fyi it's better practise to use relative links (ie not using the full https:// at the start), as these will load more quickly and will work if you're working locally on the site.